draft-irtf-cfrg-frost.md

```diff
@@ -1204,11 +1204,39 @@ T. Wilson-Brown,
 and Conrado Gouvea
 for their inputs and contributions.
 
-# Schnorr Signature Verification for Prime-Order Groups {#prime-order-verify}
+# Schnorr Signature Generation and Verification for Prime-Order Groups {#prime-order-verify}
```

Comment:
👍

Reply:
Any enthusiasm for qualifying these routines (or at least the signing routine) as 'singleton' or using a single, fully-capable keypair, to distinguish with threshold? I think the context implies it but I figure I'd ask.

Reply:
I'm not sure I understand the suggestion -- can you clarify?

Reply:
Suggested this tweak:
https://github.com/cfrg/draft-irtf-cfrg-frost/pull/293/files#r980463725